src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1534,6 +1551,7 @@ public enum LegendMaker implements LogicCardInfo {
           text "30 damage. Before doing damage, discard all Trainer cards attached to the Defending Pokémon."
           energyCost D, C
           onAttack {
+            //TODO: Check this not discarding Fossil placed as Pokémon.
```

Comment:
Fossils should be changed implementation to Pokémon and should not be affected by this.